README.md

```diff
@@ -42,10 +42,12 @@ To run the app from source, just clone the code and run this in your terminal:
 
 ```
 npm install
-npm run build
```

Comment:
Actually, it's one of the 2 effects of `prepublish` that I am leveraging here (and that's how smart npm is :-)):

> prepublish: Run BEFORE the package is published. (Also run on local `npm install` without any arguments.)

That's really cool because compiled files are right built right before publication time when it's about to get sent to npm, and whenever you run `npm install`, that is when you download the repository and start working on it. I tried locally and it does exactly what it says :-)